docs/about/getting-started.md

```diff
@@ -62,7 +62,7 @@ optional). Host and port are taken from `target-grpc-host` and
 
 Mittens allows you to use special keywords if you need to generate randomized urls.
 The following are available:
-- `{$currentDate|days+x,months+y,years+z}`: you can adjust the temporal offset by adding or subtracting days, months, or years. The offsets are optional and can be removed.
+- `{$currentDate|days+x,months+y,years+z,format=yyyy-MM-dd}`: you can adjust the temporal offset by adding or subtracting days, months, or years. Make sure you use the same order for each of the modifiers (ie, days must be left of months). You can optionally specify a custom format using yyyy or yy to represent the year, MM or MMM for the month and dd or d for the day.
```

Comment:
I am not sure I understand this part
`
Make sure you use the same order for each of the modifiers (ie, days must be left of months)
`

Reply:
That means that this works:

`{$currentDate|days+x,months+y}`

But this doesn't:

``{$currentDate|months+y,days+x}``

(this is valid for all params)

Reply:
Ah I see, maybe my english, but maybe this could be rephrase to something like:
```
The offsets are optional and can be removed, but their order cannot change (i.e. `days` is always first, followed by `months` then `years`).
```

Reply:
I added a suggestion for this.